scripts/pycodestyle_on_repo.py

```diff
@@ -33,7 +33,8 @@ def main():
     os.chdir(git_root)
     candidates, _ = get_affected_files()
     python_files = [
-        candidate for candidate in candidates if candidate.endswith('.py')]
+        candidate for candidate in candidates
+        if candidate.endswith('.py') and os.path.exists(candidate)]
```

Comment:
What is a case where `candidate` wouldn't exist? I'm thinking this would be more appropriate in `get_affected_files()`?